parts/action/query-grid.html

```diff
@@ -0,0 +1,34 @@
+<!-- wp:query { "query":{ "inherit": true }, "displayLayout": { "type": "flex", "columns":4 } } -->
```

Comment:
I added these new templates with the main reason to differentiate post and action listing templates.
We'll potentially add the CTA into action templates, but not now since it's an MVP.